src/main/kotlin/calculator/domain/Term.kt

```diff
@@ -0,0 +1,16 @@
+package calculator.domain
+
+@JvmInline
+value class Term private constructor(val value: Int) {
+
+    constructor(input: String) : this(input.toPositiveOrThrow())
+
+    companion object {
+        @JvmStatic
+        private fun String.toPositiveOrThrow(): Int {
+            val value = toIntOrNull() ?: throw RuntimeException("숫자가 아닙니다.")
+            require(value >= 0) { "음수는 입력할 수 없습니다." }
+            return value
+        }
+    }
```

Comment:
`value class`를 잘 사용해주신 것 같아요👍👍

하지만 주생성자가 private인데 부생성자는 public으로 열어둔 부분이 조금 어색하다고 느껴질 수 있을 것 같아요.

`Term` 객체를 생성할 때 음수를 가지지 않는다는 것을 보장하는 목적이라면 `init`을 통해 검증하면 어떨까요?

혹은 생성자를 모두 숨기고 팩토리 함수만 제공하는 방식도 괜찮을 것 같아요🙂
```suggestion
@JvmInline
value class Term (val value: Int) {

    constructor(input: String) : this(
        input.toIntOrNull() ?: throw IllegalArgumentException("숫자가 아닙니다.")
    )
    
    init {
        require(value >= 0) { "음수는 입력할 수 없습니다." }
    }
}
```